internal/commands/app/create_inputs.go

```diff
@@ -108,13 +108,16 @@ func (i *createInputs) resolveLocalPath(ui terminal.UI, wd string) (string, erro
 	if !fi.Mode().IsDir() {
 		return fullPath, nil
 	}
-	_, appOK, err := local.FindApp(fullPath)
+	_, _, err = local.FindApp(fullPath)
 	if err != nil {
 		return "", err
 	}
-	if appOK {
-		return "", errProjectExists{fullPath}
+
```

Comment:
instead of returning error due to existing path, prompt for alternative path

Reply:
I think we might still want this check?  What this prevents is a user from creating an app directory inside another app directory.  I think thats something we still can enforce for now, but i might also be missing something here.  let me know what you think

Reply:
is that specific case being check [here](https://github.com/10gen/realm-cli/blob/7437d6edb804f7700b4cd92ffbf9dfb3cddd0bcf/internal/commands/app/init_inputs.go#L21-L23)?

Reply:
I think that link you provided is doing the same check, yes.  Those are for the `app init` inputs though, so something similar would still need to live here as well (unless I'm missing something here)

And while I actually really like the idea of having that check again, I think the "true fix" there would be to say: "let's support auto-complete in a way that the user's input can be tabbed here and the list of matching directories (if any) would show"

Something we can definitely punt on for now.  A lot of words for basically saying "great idea, yes we'd want that, but we'll tackle it in a different ticket" :)

Reply:
Could you please explain the auto-complete idea? 

I think the errProjectExists is a good catch here in case the user, when prompted, inputs a local path that already exists.

Reply:
Yeah so definitely don't worry about the auto-complete here, but the idea is similar to when you're working in a terminal, and say you try to `cd` into a directory.  As you type out a name, you can hit "tab" and the shell generally will either auto-fill the name to what it knows is present, and/or show you a list of matches afterwards.

That way, the user is given more immediate feedback about the directories that are present on disk at the time, rather than typing in a name, hitting enter, and letting another cycle of the same validations run.

Again, I think what we have here with the first initial check is fine.  No need to over-complicate it with recursive validations or auto-complete just yet.